docs/configuration/cluster_manager/cluster_runtime.rst

```diff
@@ -30,7 +30,8 @@ Outlier detection
 
 See the outlier detection :ref:`architecture overview <arch_overview_outlier_detection>` for more
 information on outlier detection. The runtime parameters supported by outlier detection are the 
-same as the :ref:`static configuration parameters <config_cluster_manager_cluster_outlier_detection>`, namely
+same as the :ref:`static configuration parameters <config_cluster_manager_cluster_outlier_detection>`, with
```

Comment:
Why not make this configurable in the config like everything else? Seems simple to do.